test/common/grpc/grpc_client_integration.h

```diff
@@ -52,21 +52,53 @@ class GrpcClientIntegrationParamTest
   ClientType clientType() const override { return std::get<1>(GetParam()); }
 };
 
+class DeltaSotwIntegrationParamTest
+    : public testing::TestWithParam<
+          std::tuple<Network::Address::IpVersion, ClientType, SotwOrDelta>> {
+public:
+  ~DeltaSotwIntegrationParamTest() override = default;
+  static std::string
+  protocolTestParamsToString(const ::testing::TestParamInfo<
+                             std::tuple<Network::Address::IpVersion, ClientType, SotwOrDelta>>& p) {
+    return absl::StrCat(
+        (std::get<0>(p.param) == Network::Address::IpVersion::v4 ? "IPv4_" : "IPv6_"),
+        (std::get<1>(p.param) == ClientType::GoogleGrpc ? "GoogleGrpc" : "EnvoyGrpc"),
```

Comment:
Nit: another `_` to separate gRPC and API type? Might be clearer actually via a `fmt::format`, but don't mind either way.

Reply:
Done.